the-quest

```diff
@@ -0,0 +1 @@
+the journey of inner peace can be thought to be located far but instead lives within
```

Comment:
the journey of inner peace IS thought to be located far but instead lives within